docs/src/app/(website)/page.tsx

```diff
@@ -0,0 +1,451 @@
+                      Base UI is a library of unstyled UI components for building accessible
+                      component libraries, user interfaces, web applications, and websites with
+                      React. Base UI components are highly configurable, composable, and
+                      customizable.
+                    </p>
+                  </Accordion.Panel>
+                </Accordion.Item>
+                <Accordion.Item className="AccordionItem">
+                  <Accordion.Header className="AccordionHeader">
+                    <Accordion.Trigger className="AccordionTrigger Text size-2">
+                      Does Base UI work with any styling library?
+                      <PlusIcon className="AccordionIcon AccordionIconPlus" />
+                      <MinusIcon className="AccordionIcon AccordionIconMinus" />
+                    </Accordion.Trigger>
+                  </Accordion.Header>
+                  <Accordion.Panel className="AccordionPanel">
+                    <p className="Text size-2">
+                      Yes. Base UI works with Tailwind, CSS Modules, CSS-in-JS, plain CSS, and any
+                      other styling library you prefer. It also works with JavaScript animation
+                      libraries like Motion, or just plain CSS transitions. Base UI is an unstyled
+                      component library. The package does not bundle any CSS, and does not prescribe
+                      any styling solution.
+                    </p>
+                  </Accordion.Panel>
+                </Accordion.Item>
+                <Accordion.Item className="AccordionItem">
+                  <Accordion.Header className="AccordionHeader">
+                    <Accordion.Trigger className="AccordionTrigger Text size-2">
+                      Which accessibility standards does Base UI follow?
+                      <PlusIcon className="AccordionIcon AccordionIconPlus" />
+                      <MinusIcon className="AccordionIcon AccordionIconMinus" />
+                    </Accordion.Trigger>
+                  </Accordion.Header>
+                  <Accordion.Panel className="AccordionPanel">
+                    <p className="Text size-2">
+                      When designing and speccing components, we follow{' '}
+                      <a className="Link" href="https://www.w3.org/WAI/ARIA/apg/patterns/">
+                        ARIA Authoring Practices Guide patterns
+                      </a>
+                      , and comply with the{' '}
+                      <a
+                        className="Link"
+                        href="https://www.w3.org/TR/WCAG22/#new-features-in-wcag-2-2"
+                      >
+                        WCAG 2.2 standard
+                      </a>
+                      . Base UI is compliant with all Success Criteria levels relating to component
+                      behaviour. However, in most cases, we go way beyond these guides. Base UI
```

Comment:
IIRC we use American spelling

```suggestion
                      behavior. However, in most cases, we go way beyond these guides. Base UI
```